onnxruntime/core/providers/js/operators/conv.h

```diff
@@ -94,5 +94,12 @@ class Conv : public JsKernel {
   // Tensor w_transposed_;
 };
 
+template <typename T, bool is_channels_last>
+class FusedConv : public Conv<T, is_channels_last> {
+ public:
+  explicit FusedConv(const OpKernelInfo& info) : Conv<T, is_channels_last>(info) {
+    ORT_ENFORCE(info.GetAttr<std::string>("activation", &(this->conv_attrs_.activation)).IsOK());
```

Comment:
Setting this attribute here will not pass the value to JSEP kernel because JSEP kernel is already created in the base class contructor.